dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/util/AlertService.java

```diff
@@ -48,8 +43,6 @@ public AlertService(final ResourceBundle resourceBundle) {
 
 
     private void showAlertInternal(final String message, final String title, final Alert.AlertType alertType, final Window window) {
-        Platform.runLater(() -> {
```

Comment:
removing that will still show the popup?

Reply:
Yeah, it still shows, but now it doesn't lock the UI.

Reply:
ohh... right this is blocking call

Reply:
It's not a blocking call.

Reply:
UI blocking

Reply:
Can't this method be called from a non-javafx thread?